src/components/IconSVG/ChangeColor.tsx

```diff
@@ -0,0 +1,16 @@
+const ChangeColor = ({ color }: { color: string }) => {
+  return (
+    <svg
+      width="41"
+      height="15"
+      viewBox="0 0 41 15"
+      fill="none"
```

Comment:
This component already exists in master version please update

Reply:
Done